package.json

```diff
@@ -4,6 +4,7 @@
   "description": "tupperware bot for discord tulpa hooks",
   "main": "bot.js",
   "scripts": {
+    "start": "node bot",
```

Comment:
This just lets you use `npm start` which is pretty standard to start servers and some tooling picks up on npm scripts